From 7597cfa482e42a00a69fb9577ee523762980a9a2 Mon Sep 17 00:00:00 2001
From: Michael Orlitzky <michael@orlitzky.com>
Date: Thu, 28 Dec 2017 15:52:42 -0500
Subject: [PATCH] doc: clarify chown/chgrp --dereference defaults

* doc/coreutils.texi: the documentation for the --dereference
  flag of chown/chgrp states that it is the default mode of
  operation. Document that this is only the case when operating
  non-recursively.
---
 doc/coreutils.texi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/doc/coreutils.texi b/doc/coreutils.texi
index 3fa083085..1c0e8a36c 100644
--- a/doc/coreutils.texi
+++ b/doc/coreutils.texi
@@ -10989,7 +10989,7 @@ chown -h -R --from=OLDUSER NEWUSER /
 @cindex symbolic links, changing owner
 @findex lchown
 Do not act on symbolic links themselves but rather on what they point to.
-This is the default.
+This is the default when not operating recursively.
 
 @item -h
 @itemx --no-dereference
@@ -11119,7 +11119,7 @@ changed.
 @cindex symbolic links, changing owner
 @findex lchown
 Do not act on symbolic links themselves but rather on what they point to.
-This is the default.
+This is the default when not operating recursively.
 
 @item -h
 @itemx --no-dereference
-- 
2.15.2

